### DISCUSSION TITLE
Is there any fair use policy/usage restrictions documentation ?

### DISCUSSION THREAD

**Discussion by amggarcia** [trakt/trakt-api#457] [Q&A]
Heyo, was wondering if there is any fair-use policy / usage restrictions documentation available ?

Searched around but could not find anything related to those topics when it comes to API usage for other projects.

To give some more context I am asking this because I'm considering building a WebbApp to use Trakt through.

As amazing as Trakt is I personally would not use 1/10th of the features it provides, i just want something really simple and "minimalist" to track my shows with.

After failing to find something to match what I'm looking for in a tracking app, I thought about building my own using Trakt api/backend as a information source.

I was just trying to find out what the fair-use/usage restrictions are for the Trakt API before going through with building something like this.

I assume that making something private that only I use is more then ok but wanted to know ahead of time if I could make this public or if there are any restrictions when it comes to making something like that available to others as well.

Thanks in advance.

**Comment by rectifyer** [comment]
A private project is fine. We wouldn't want a public project using the Trakt API that recreates what Trakt is already doing on the website and apps.

**Comment by amggarcia** [comment]
Figured that would be the case hence why I asked, and it makes perfect sense.

Just a followup question though, If that is the case, do mobile (android/ios) apps consuming the Trakt API fall under a different scenario/scope ? 

Since even on the https://trakt.tv/apps page many different ones are presented side-by-side with the official Trakt App ?
